debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/CoreInfoCommand.java

```diff
@@ -105,7 +105,9 @@ public void run(String command, String[] args, Context context, PrintStream out)
 			/* Print Java Version Info */
 			out.println("JAVA VERSION INFO\n" + properties.get("java.fullversion"));
 			/* Print Java VM Version Info */
-			out.println("JAVA VM VERSION\t- " + properties.get("java.vm.version") + "\n");
+			out.println("JAVA VM VERSION\t- " + properties.get("java.vm.version"));
+			/* Print Java VM Name */
+			out.println("JAVA VM NAME\t- " + properties.get("java.vm.name"));
```

Comment:
There used to be a blank line between the JAVA SERVICE LEVEL INFO section and the PLATFORM INFO section, but with the removal of the ` + "\n"` that blank line is gone.

Reply:
I also note the pre-existing code using `\n` isn't correct. It should be using `System.getProperty("line.separator")` or `out.println();`.

Reply:
I'll open a pull request to clean that up later today.

Reply:
@dusanboskovic was working on something, pls check with him.

Reply:
@keithc-ca Peter and I were speaking about this last night. I can open a new PR today that will clean up the "\n" and reintroduce a new blank line after the "JAVA VM NAME" section

Reply:
I don't think the blank line after 'JAVA VM NAME' is significant; I think it's just a stylistic preference (that I'm fine with either way).

Reply:
There were/are sections in the output which are separated with blank lines. The blank line between two sections was removed, while there is still a blank line between other sections. I think removing the blank line makes the output a little harder to read. 

This is what it looks like now, without a blank line before `PLATFORM INFO`.
```
COMMANDLINE
bin/java -Xdump:system:events=vmstop

JAVA SERVICE LEVEL INFO
JRE 1.8.0 Linux amd64-64 (build 8.0.7.0 - pxa6480sr7-20200922_01(SR7))
JAVA VERSION INFO
JRE 1.8.0 Linux amd64-64-Bit Compressed References 20201006_456922 (JIT enabled, AOT enabled)
OpenJ9   - a12d134
OMR      - 738adf7
IBM      - 712c984
JAVA VM VERSION	- 2.9
JAVA VM NAME	- IBM J9 VM
PLATFORM INFO
Platform Name :	LINUX 64Bit
OS Level	: Linux 3.10.0-693.1.1.el7.x86_64
Processors -
  Architecture	: amd64
  How Many	: 2
```

Reply:
I think a blank line would be (more) welcome before 'JAVA VM VERSION' to make the end of 'JAVA VERSION INFO' more visible.

Reply:
I am looking at it now, an I think it would be best if it was reordered. Adding a blank line before 'JAVA VM VERSION' makes it seem chopped. How does something like this look
```
JAVA VM NAME	- IBM J9 VM
JAVA VM VERSION	- 2.9
JAVA SERVICE LEVEL INFO
JRE 1.8.0 Linux amd64-64 (build 8.0.7.0 - pxa6480sr7-20200922_01(SR7))
JAVA VERSION INFO
JRE 1.8.0 Linux amd64-64-Bit Compressed References 20201006_456922 (JIT enabled, AOT enabled)
OpenJ9   - a12d134
OMR      - 738adf7
IBM      - 712c984
PLATFORM INFO
Platform Name :	LINUX 64Bit
OS Level	: Linux 3.10.0-693.1.1.el7.x86_64
Processors -
  Architecture	: amd64
  How Many	: 2
```

Reply:
It's fine with me.

Reply:
New PR: https://github.com/eclipse/openj9/pull/10827